appclient/src/test/resources/org/jboss/test/metadata/appclient/AppClient10Everything_testEverything.xml

```diff
@@ -638,4 +638,129 @@
         <resource-adapter>appAdministeredObject3ResourceAdapter</resource-adapter>
     </administered-object>
 
+    <!-- jakartaee:10:context-service -->
```

Comment:
Similar to another comment on the review, I don't get how these elements are made available for this XSD. I suspect they are not available for application client descriptors.

Reply:
fixed